tests/unit/amo/pages/TestCategory.js

```diff
@@ -749,4 +750,39 @@ describe(__filename, () => {
       expect(root.find(NotFound)).toHaveLength(0);
     });
   });
+
+  it('renders a 404 when clientApp is Android and enableFeatureStaticThemesForAndroid is false', () => {
+    store = dispatchClientMetadata({ clientApp: CLIENT_APP_ANDROID }).store;
+    const _config = getFakeConfig({
+      enableFeatureStaticThemesForAndroid: false,
+    });
+
+    const root = render({ _config, store });
+
+    expect(root.find(NotFound)).toHaveLength(1);
```

Comment:
should we add this too: `expect(root).toHaveClassName('Category');`

Reply:
sorry I meant `expect(root).not.toHaveClassName('Category');` or `expect(root.find('Category')).toHaveLength(0);`